### ISSUE TITLE
Cannot install any dependencies in element-lib

### ISSUE THREAD

**Issue by gjgd** [decentralized-identity/element#48]
A faulty version of websocket was released (.git included in the package) causes an error when `npm i --save <any-dependency>` 

Example:
```
➜ npm i rxjs --save
npm ERR! path /Users/guillaume/workspace/work/elem/packages/element-lib/node_modules/web3/node_modules/websocket
npm ERR! code EISGIT
npm ERR! git /Users/guillaume/workspace/work/elem/packages/element-lib/node_modules/web3/node_modules/websocket: Appears to be a git repo or submodule.
npm ERR! git     /Users/guillaume/workspace/work/elem/packages/element-lib/node_modules/web3/node_modules/websocket
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/guillaume/.npm/_logs/2019-08-01T11_20_08_862Z-debug.log
```

The issue was solved here in v1.0.29: https://github.com/theturtle32/WebSocket-Node/blob/master/CHANGELOG.md#version-1029

We have 2 sub dependencies who use `websocket`:
- `web3-providers-ws`
- `web3`

**Comment by gjgd** [comment]
We need to wait for those dependencies to update to 1.0.29 to get rid of the error.

In the meantime we can run`rm -rf node_modules/web3-providers-ws/node_modules/websocket/.git node_modules/web3/node_modules/websocket/.git node_modules/web3/node_modules/web3-providers-ws/node_modules/websocket/.git
` on pre-install

**Comment by gjgd** [comment]
This issue will be solved in web3@1.2.1, that will be released in a few days so let's wait until then to solve this issue

https://github.com/ethereum/web3.js/issues/3002#issuecomment-518155102
